src/components/ContactInput.tsx

```diff
@@ -17,28 +17,29 @@ interface IProps {
 
 const ContactInputComponent = ({
   disabled = false,
-  label="Phone Number",
+  label = "Phone Number",
   className,
   zipClass,
   prefix,
 }: IProps) => {
   const { countries } = useCountryStore();
   const [filteredCountries, setFilteredCountries] = useState<countryType[]>([]);
   const [searchTerm, setSearchTerm] = useState("");
+  const [isDropdownOpen, setIsDropdownOpen] = useState(false);
+  const { setFieldValue, errors, touched } = useFormikContext<object>();
 
-  const { setFieldValue } = useFormikContext<object>();
-
-  // Sync search term with zip code prop updates
-  // useEffect(() => {
-  //   if (!searchTerm) setSearchTerm(zipCode || "");
-  // }, [zipCode, setSearchTerm]);
+  // Sort countries alphabetically by name
+  const sortedCountries = [...countries].sort((a, b) => 
```

Comment:
sorting it before saving it in the store will provide better performance 
this way every instance of the component will cause the sorting and causing a higher cost for computing